projects/mci/inc/regen_braking.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
```

Comment:
Could you include a comment here with a brief description of what the module does, and then what libraries it needs to be initialized? Like `// Requires x, y, and z to be initialized.`. All our head usually have that kind of documentation. (This one especially I think you should mention that this module is for managing whether or not we're currently regen braking, not for actually setting the current/velocity related to regen braking.)

Reply:
Done